network-api/networkapi/templates/fragments/blocks/nav/item.html

```diff
@@ -0,0 +1,17 @@
+{% load static i18n wagtailadmin_tags %}
+
+{% fragment as font_styles %}tw-text-xl tw-font-semibold tw-capitalize tw-text-black{% endfragment %}
+{% fragment as hover_styles %}group-hover:tw-text-blue-80 group-hover:tw-underline{% endfragment %}
+{% fragment as after_arrow_styles %}after:tw-content-['_↗'] after:tw-font-['Nunito_Sans'] after:tw-text-gray-20 group-hover:after:tw-text-blue-80{% endfragment %}
```

Comment:
```suggestion
{% fragment as after_arrow_styles %}after:tw-content-['_↗'] after:tw-font-sans after:tw-text-gray-20 group-hover:after:tw-text-blue-80{% endfragment %}
```

Reply:
The glyphs do not work properly with `tw-font-sans`. It seems that one of the other fonts takes precedence and then the arrow is displayed weirdly:

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/61277874/0f54aa46-8b9e-441f-b440-d29e3a74414a)

Forcing "Nunito Sans" (using `tw-font-['Nunito_Sans']`), we get the expected result:

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/61277874/81d56fbc-158a-489a-b75b-ec5b7492b567)

I can add a comment if it helps...

Reply:
Ah I see. Don't worry about this now. We can do a final check at a later phase prior to requesting review from designers.